js/interactive.js

```diff
@@ -456,6 +459,9 @@ var source_list = new Map([
         return `${acc}${ite}`;
       }
       $("div.misinfo-response-area").html(doms.reduce(red));
+      $("#misinformation-info > div.info.misinfo-response-area > div.misinfo-block > div.metrics").click(function() {
+        console.log("extract 'data-container' and parse here, then send 2 required parameters to /mtweets endpoint and render response here too.");
```

Comment:
Will need to construct DOM to append to existing DOM tree, and we will need some `.css` work to make it look good as well.

The desired look should be a similar [modal](https://getbootstrap.com/docs/4.0/components/modal/) to the one that appears when clicking the `?` on the website (for consistency).